woocommerce/payment-gateway/assets/css/frontend/sv-wc-payment-gateway-payment-form.scss

```diff
@@ -118,19 +118,16 @@ http://www.gnu.org/licenses/gpl-3.0.html
 	}
 }
 
-// add payment method form card images
-#add_payment_method {
-	ul.payment_methods li {
-		list-style-type: none;
+// payment form card images
+ul.payment_methods li {
+
+	div {
```

Comment:
we should add a class to target for this - this will apply to _any_ div within the payment method UI, so things like the payment form itself get the styling applied which causes unexpected results

Reply:
Makes sense! Fixed in bf257f9.